corehq/apps/data_dictionary/views.py

```diff
@@ -384,9 +426,10 @@ def _process_bulk_upload(bulk_file, domain):
                         if fhir_resource_prop_path and not fhir_resource_type:
                             error = _('Could not find resource type for {}').format(case_type)
                     if not error:
+                        allowed_values = allowed_value_info[case_type][name]
```

Comment:
this errors if the property does not exist in the dictionary

Reply:
Yes I noticed that too when looking into QA-3457 and plan to fix both issues.